packages/@adobe/spectrum-css-temp/components/colorwheel/skin.css

```diff
@@ -5,13 +5,13 @@
 
 .spectrum-ColorWheel {
   &.is-disabled {
-    .spectrum-ColorWheel-outerCircle,
-    .spectrum-ColorWheel-innerCircle {
-      stroke: var(--spectrum-colorwheel-border-color-disabled);
-    }
+    .spectrum-ColorWheel-gradient {
+      background: var(--spectrum-colorwheel-fill-color-disabled) !important;
```

Comment:
same thing, do we just need higher specificity? or what's going on?

Reply:
the gradient is on an inline style.

Reply:
hmmm could that inline style be a variable that we could apply here as multiple backgrounds?
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Backgrounds_and_Borders/Using_multiple_backgrounds

or should disabled also be inline?

i'll keep thinking

Reply:
yes, or we could not set it when disabled. however, this is all spectrum specific and the background is generated in aria...